bok_choy/a11y/a11y_audit.py

```diff
@@ -0,0 +1,253 @@
+"""
+Interface for running accessibility audits on a PageObject.
+"""
+import json
+import os
+import requests
+
+from textwrap import dedent
+
+
+class AccessibilityError(Exception):
+    """
+    The page violates one or more accessibility rules.
+    """
+    pass
+
+
+class A11yAuditConfig(object):
+    """
+    The `A11yAuditConfig` object defines the options available in an
+    accessibility ruleset.
+    """
+    def __init__(self, *args, **kwargs):
+        super(A11yAuditConfig, self).__init__(*args, **kwargs)
+        self.rules_file = None
+
+    def set_rules_file(self, path=None):
+        """
+        Sets `self.rules_file` to the passed file.
+
+        Args:
+            A filepath where the JavaScript for the ruleset can be found.
+
+        This is intended to be used in the case of using an extended or
+        modified version of the ruleset. The interface and response
+        format are expected to be unmodified.
+        """
+        if path:
+            self.rules_file = os.path.abspath(path)
+
+    def set_rules(self, rules, ignore=False, include_type="rule"):
```

Comment:
Nice catch.  They don't need it necessarily, but it's nice.  I just updated this and a couple of other methods that should have it too.